### ISSUE TITLE
Undying and invulnerable

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#6403]
(WEB REPORT BY: killette2 REMOTE: 158.69.55.2:7777)
Get cloned, succumb while in the cloner. Normally this makes you die, but I came out standing up with negative damage. A doctor then tried to beat me to death but it didn't work, admin intervention was needed. I couldn't interact with any objects or people, but I could push and pull them around with my body.

**Comment by FudgePucker** [comment]
I guess preventing succumbing a while after being cloned would fix this.

**Comment by PJB3005** [comment]
I think we're looking at a bigger issue than succumbing in cloning = issue.

**Comment by FudgePucker** [comment]
My guess is that it has to do with how cloners work, succumbing while in the early crit stages sets you to dead, the cloner brings you out of crit, giving you back control, but you are still dead.

Just preventing succumbing while in a cloner would solve it. Sure there might be a bigger issue at play here, but you are forgetting one thing. The only way that a dead player can be healed is from the cloner, all other ways that a player can leave critical is from chemicals, players stop processing chemicals when they are dead, if you succumb in cyro it stops healing you because you are dead, chemicals stop working.

In summary, cloners are the only way a dead player can be healed, rather than looking for a needle in a haystack, just prevent succumbing in a cloner. The only reason you would succumb in a cloner is to griff.

**Comment by d3athrow** [comment]
I think this and #6404 were related

**Comment by clusterfack** [comment]
Issue disappeared with #6404
